proto/panacea/oracle/v2alpha2/genesis.proto

```diff
@@ -10,6 +10,11 @@ import "panacea/oracle/v2alpha2/oracle.proto";
 
 // GenesisState defines the oracle module's genesis state.
 message GenesisState {
+  repeated Oracle oracles = 1 [(gogoproto.nullable) = false];
+  repeated OracleRegistration oracle_registrations = 2 [(gogoproto.nullable) = false];
+  repeated OracleRegistrationVote oracle_registration_votes = 3 [(gogoproto.nullable) = false];
+  Params params = 4 [(gogoproto.nullable) = false];
+  UpgradeOracleInfo upgrade_oracle_info = 5 [(gogoproto.nullable) = false];
```

Comment:
i think it's better to rename this to OracleUpgradeInfo.

Reply:
Thank you. I fixed it.(https://github.com/medibloc/panacea-core/pull/366/commits/921e33050dbc6008c5e3f7ccf2abdf03b3b2c86e)